bioptim/limits/path_conditions.py

```diff
@@ -401,6 +401,19 @@ def concatenate(self, other: "Bounds"):
         self.extra_params = self.min.extra_params
         self.n_shooting = self.min.n_shooting
 
+    def scale(self, scaling: Union[float, np.ndarray]) -> "Bounds":
+        """
+        Scaling a Bound
+
+        Parameters
+        ----------
+        scaling: float
+            The scaling factor
+        """
+        self.min = self.min / scaling
+        self.max = self.max / scaling
```

Comment:
It is fine like this, you can also consider the `/=` . I am not sure if it is optimized (compared to self = self / val).. In doubt I'd used the /= but it is probably marginally different

Reply:
If the scaling is done internally (i.e., the user is not expected to call the scaling method by themselves), maybe we should consider raising a ValueError if the dimensions of the vectors do not agree to pass the information to the user that it fails because of the scaling (and not just a generic message of shapes don't match)?